Source: Hoshpak
Upstream: https://github.com/sddm/sddm/issues/903 and https://github.com/sddm/sddm/pull/923
Reason: removes detection for the ConsoleKit interface because the ConsoleKit2 code path is 
 currently broken and would cause sddm to fail to start with ConsoleKit2 present
--- a/src/daemon/LogindDBusTypes.cpp	2018-10-07 17:40:49.944625726 +0200
+++ b/src/daemon/LogindDBusTypes.cpp	2018-10-07 17:41:01.590479575 +0200
@@ -58,17 +58,6 @@
         return;
     }
 
-    if (QDBusConnection::systemBus().interface()->isServiceRegistered(QStringLiteral("org.freedesktop.ConsoleKit"))) {
-        qDebug() << "Console kit interface found";
-        available = true;
-        serviceName = QStringLiteral("org.freedesktop.ConsoleKit");
-        managerPath = QStringLiteral("/org/freedesktop/ConsoleKit/Manager");
-        managerIfaceName = QStringLiteral("org.freedesktop.ConsoleKit.Manager"); //note this doesn't match logind
-        seatIfaceName = QStringLiteral("org.freedesktop.ConsoleKit.Seat");
-        sessionIfaceName = QStringLiteral("org.freedesktop.ConsoleKit.Session");
-        userIfaceName = QStringLiteral("org.freedesktop.ConsoleKit.User");
-        return;
-    }
     qDebug() << "No session manager found";
 }
 
